hw/syn/tools/dc/run-syn.tcl

```diff
@@ -63,12 +50,12 @@ set PARAMS ""
 remove_design -designs
 sh rm -rf $WORKLIB/*
 
-analyze -define ${DEFINE} -format sv ${SRC}
-elaborate  ${TOP_ENTITY} -parameters ${PARAMS}
-link > "${REPDIR}/${TOP_ENTITY}_link.rpt"
+analyze -vcs "-sverilog +define+${DEFINE} -f ${SV_FLIST}" > "${REPDIR}/analyze.rpt"
+elaborate  ${DUT} -parameters ${PARAMS}                   > "${REPDIR}/elab.rpt"
```

Comment:
@sriyerg this is where the DUT name is still needed. I have removed it from the report names, since the build dir is already differentiated for each DUT.

Reply:
ok sounds good.